civictechprojects/static/css/partials/_Profile.scss

```diff
@@ -126,7 +126,7 @@
     flex-basis: auto;
     img {
       width: 100%;
-      max-height: unset;
+      max-height: 250px;
```

Comment:
This looks good, but as one last polish item -- we can just remove this line entirely. 

If you look at L21, max-height 250 is already declared so if we just remove this rule entirely it'll fall back on that and not require re-declaring it here.

![image](https://user-images.githubusercontent.com/16870466/132767064-120baf16-1579-493e-a5b3-616138d0a6a6.png)
As you can see from the element inspector, this max-height in line 129 is overriding the prior one - and since they're the same number we may as well just do it once. 

Otherwise good fix, looks like the merge worked great, ready for approval as soon as you get that in.

Reply:
Hi I made the change requested